game-rules.md

```diff
@@ -75,29 +72,31 @@ The building will be a lower-case letter if it is still under construction and i
 ### Command Format:
 * The command format is '**x,y,building_type**'.
 * The input must be comma-separated numbers that represent the coordinates followed by the numeric ID of the building type.
-* The valid building types are:
-    * 0: **Defence Building**
-    * 1: **Attack Building**
-    * 2: **Energy Building**
+* The valid commands are:
+    * 0: Build a **Defence Building**
+    * 1: Build a **Attack Building**
+    * 2: Build a **Energy Building**
+	* 3: **Deconstruct** a building
+	* 4: Build a **Tesla Tower**
 * An example of a valid command will be '**0,0,1**', which will place an attack building at the top left corner of the map.
 
 ## Buildings:
 * Each building will have an energy cost to build.
 * Each building will take a number of turns until it is fully built and ready, based on its type.
-
-* Buildings that are busy being built will not be hit by missles travelling over it.
-
+* Buildings that are busy being built will not be hit by missiles or lightning over it.
 * The energy cost of a building is deducted from a player's energy total once the command to build a building has been successful.
 * Trying to place a building without enough energy for that building will result in a **'Do nothing'** command.
-* Once a building is destroyed, another building can be placed at that location without penalty.
-* Only one building of any type can occupy a location on the map.
+* Once a building is destroyed, another building can be placed at that location.
+* Only one building can occupy a location on the map.
+* Damaging enemy buildings will award score to you, depending on the building type.
+* Constructing new buildings will award score to you, depending on the building type.
 
 ### Building Types:
-**The player can build one of 3 different types of buildings**
+**The player can build one of 4 different types of buildings**
 
 #### Defence Building:
-A defensive building, that is tougher than any other building type with no offensive capabilities.
-Able to take more hits than any other building, these buildings are great for soaking up damage while your offensive buildings fire from safety behind them.
+A defensive building, tougher than any other building type with no offensive capabilities.
+It is able to take more hits than any other building, these buildings are great for soaking up damage while your offensive buildings fire from the safety behind them.
```

Comment:
these buildings are great for soaking up damage while your **offensive buildings fire safely from behind them**